core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -453,6 +504,25 @@ private[spark] class TaskSchedulerImpl(
     return tasks
   }
 
+  private def createUnschedulableTaskSetAbortTimer(
+      taskSet: TaskSetManager,
+      taskIndex: Int,
+      timeout: Long): TimerTask = {
+    new TimerTask() {
+      override def run() {
+        if (unschedulableTaskSetToExpiryTime.contains(taskSet) &&
+          unschedulableTaskSetToExpiryTime(taskSet) <= clock.getTimeMillis()
+        ) {
```

Comment:
nit: I think the style is to double-indent the second line of the continuation, and close it at the end of that line

```scala
if (unschedulableTaskSetToExpiryTime.contains(taskSet) &&
    unschedulableTaskSetToExpiryTime(taskSet) <= clock.getTimeMillis()) {
  logInfo
```